kubernetes/k8s-gencfg

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python -u
+"""
+Generate K8s config from template files
+"""
+
+import argparse
+import os
+import re
+import subprocess
+
+def parse_file(args, input_file, output_file):
```

Comment:
I'd argue this does more than just parse, more modifies. Don't really care though